api/v1alpha1/servicedefaults_types.go

```diff
@@ -44,9 +44,46 @@ type ServiceDefaultsStatus struct {
 type ServiceDefaults struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
+	Status            ConfigEntryStatus   `json:"status,omitempty"`
+	Spec              ServiceDefaultsSpec `json:"spec,omitempty"`
```

Comment:
I couldn't pull `metav1.TypeMeta`, `metav1.ObjectMeta` and `Status` into a shared struct that is embedded into all config entries without breaking code generation so unfortunately we'll need to implement the `ConfigEntryCRD` interface for all config entries.